src/applications/mhv-medical-records/reducers/labsAndTests.js

```diff
@@ -38,11 +39,27 @@ const initialState = {
   labsAndTestsDetails: undefined,
 };
 
+const getLabLocation = (performer, record) => {
+  if (isArrayAndHasItems(performer)) {
+    const locationRef = performer[0]?.reference;
+    const labLocation = extractContainedResource(record, locationRef);
+    return labLocation?.name;
+  }
+  return null;
+};
+
+const distilChemHemNotes = (notes, valueProp) => {
```

Comment:
Distill has two l's 🙂

Reply:
Apparently it can also have one l, but it kind of looks weird to me.

Reply:
Oh!